package.json

```diff
@@ -50,6 +50,7 @@
     "axios": "^1.6.7",
     "cmdk": "^1.1.1",
     "convex": "^1.17.2",
+    "convex-oss-stats": "link:../../../erquhart/convex-oss-stats",
```

Comment:
Just leaving this common here, so the local dependency link doesn't get shipped.

Reply:
Gah, good catch